tests/testing/BaseTestHelperTest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from coalib.results.Result import Result
+from coalib.testing.BaseTestHelper import BaseTestHelper
+
+
+class BaseTestHelperTest(BaseTestHelper):
+    def test_good_assertComparableObjectsEqual(self):
```

Comment:
A uniform case would be a lot better. `test_good_assert_comparable_objects_equal`

Reply:
same for me it should be `test_good_assertComparableObjectsEqual` as IMO `assertComparableObjectsEqual` is a method name which together means something